### PR TITLE
chore: add externalTrafficPolicy for apisix gateway service

### LINKED ISSUE #88: Apisix Gateway Ingress ```externalTrafficPolicy``` missing setting
Currently, you are unable to change ```externalTrafficPolicy``` via the apisix helm chart for the [service gateway template](https://github.com/apache/apisix-helm-chart/blob/master/charts/apisix/templates/service-gateway.yaml).

This is useful if you want to change from ```Cluster``` to ```Local```

### PR DESCRIPTION
**Pull request by tao12345666333**
Signed-off-by: Jintao Zhang <zhangjintao9020@gmail.com>

closes #88

### CODE REVIEW

File: charts/apisix/templates/service-gateway.yaml [hunk 1]
```diff
@@ -28,6 +28,7 @@ metadata:
     {{- include "apisix.labels" . | nindent 4 }}
 spec:
   type: {{ .Values.gateway.type }}
+  externalTrafficPolicy: {{ .Values.gateway.externalTrafficPolicy }}
```
Comment (tokers) [inline charts/apisix/templates/service-gateway.yaml:31]: We can set this field only if the service type is `LoadBalancer`.

Comment (tao12345666333) [reply charts/apisix/templates/service-gateway.yaml:31]: Both `LoadBalancer` and `NodePort` are affected by this parameter.

Comment (tao12345666333) [reply charts/apisix/templates/service-gateway.yaml:31]: Both `LoadBalancer` and `NodePort` are affected by this parameter. 
I don’t think it is necessary to add judgment here.

**Review by gxthrj — APPROVED** [review]

File: charts/apisix/templates/service-gateway.yaml (hunk 1 above)
Comment (tokers) [reply charts/apisix/templates/service-gateway.yaml:31]: Got it.

**Review by tokers — APPROVED** [review]

**Pull request merged**